### ISSUE TITLE
Version 2

### ISSUE THREAD

**Issue by loeffel-io** [loeffel-io/ls-lint#36]
Thank you for 100 000 NPM Downloads! The project is growing constantly and we are starting the development of v2 soon!

Planned features: 

- Extend `|` operator with a ongoing and trailing whitespace to support the regex alternation operator #34 ✅ 
- `exists` + `not_exists` + `only` directory rules #30 #32 (for example: `only:.js,.ts`)
- Support `.yaml` configs #29 ✅ 
- Glob support for `ignore` #35 
- Reusable rules #22
- Performance optimizations ✅ 
- Support arm arch ✅ 
- Support for SCREAMING_SNAKE_CASE ✅ 
- ls-lint on an existing project ("warn" option) #63 ✅ 
- https://github.com/loeffel-io/ls-lint/issues/71

**Comment by superlbr** [comment]
hi, any plan to support arm?
```
ls-lint@1.9.2: The CPU architecture "arm64" is incompatible with this module
```

**Comment by loeffel-io** [comment]
Yes, added to the list :+1:

**Comment by loeffel-io** [comment]
Pretty cool idea: #40

**Comment by loeffel-io** [comment]
Added: #44

**Comment by wiget** [comment]
Arm support is in #43

**Comment by jnardone** [comment]
Any ETA on this?  The glob stuff alone for `ignore` is killing me.

**Comment by thescientist13** [comment]
Looks like as of version >= 1.10.0 arm64 M1 Mac support Is available.  Thank you!  🙏

**Comment by loeffel-io** [comment]
https://github.com/loeffel-io/ls-lint/pull/66 is on the way with some great changes 🕺

**Comment by hanneskaeufler-bmw** [comment]
Is there any plan for a date for the V2 release? We are very interested in the `ignore` globbing, but unsure if that feature is near. Would be cool if anyone has any insight on the date or whether that feature is already implemented somewhere in a branch maybe? Thanks!

**Comment by loeffel-io** [comment]
Hey @hanneskaeufler-bmw, 

i really hope to get to this asap - feel free to submit a pr! ❤️

**Comment by levaleks** [comment]
Hey @loeffel-io Thanks for the great lib!
Any updates on v2 release?

**Comment by jiaxp** [comment]
Any updates on v2 release?

**Comment by gassims** [comment]
> hi, any plan to support arm?
> 
> ```
> ls-lint@1.9.2: The CPU architecture "arm64" is incompatible with this module
> ```

We've got the same issue with "ia32"? @loeffel-io  is that something you could support please. I'm using an MSI laptop with a 64-bit Windows 10 system (https://github.com/loeffel-io/ls-lint/issues/80)

Thanks!

**Comment by loeffel-io** [comment]
Very happy to announce that `v2.0.0-beta.0` is now available: https://github.com/loeffel-io/ls-lint/releases/tag/v2.0.0-beta.0
Please see the changelog for any additional info: https://ls-lint.org/2.0/prologue/changelog.html#v2-0-0-beta

Feel free to try it out

**Comment by loeffel-io** [comment]
v2.0.0 is officially released! 🎉 

I close this issue - there will be a follow up roadmap ticket in the next weeks

**Comment by loeffel-io** [comment]
I created a roadmap for the next planned features #122

**Comment by loeffel-io** [comment]
The glob ignore feature is now available in beta: ´v2.1.0-beta.0`

**Comment by loeffel-io** [comment]
`v2.2.0-beta.0` is now available 🎉 
https://ls-lint.org/2.2/prologue/changelog.html#v2-2-0-beta-0
